### PR TITLE
docs: Add 4intelligence

### PR DESCRIPTION
**Pull request by thiago4int**

### CODE REVIEW

File: .dccache [hunk 1]
```diff
@@ -0,0 +1 @@
...b45de452b3fcfd6e7432cda7dc9c285f92af4efb7d"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflow-timeline/workflow-timeline.tsx":[6668,1626869650853.286,"e5317eae47cc09e974fe50fb86159599e0a95f41a9e303eb177afb967db3ace4"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflow-yaml-viewer/workflow-yaml-viewer.tsx":[2712,1626869650853.286,"063b7bfa01df6218c1c03ae30f6404fc45eb2f494f13c2cbe610fc65053d0f45"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflows-list/workflows-list.tsx":[20234,1626869650853.286,"e8ef1001ff9dcb6e44613f6589c7957cc69e9562b300a0cd092fd6a75ed0a015"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflows-row/workflows-row.tsx":[4632,1626869650853.286,"a6375beca1daa4ed369d7ecb1050fd885c46bb8283bb1316bc1bdf9e35b7180c"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflows-toolbar/workflows-toolbar.tsx":[4572,1626869650853.286,"984f1f98ee7e86ce7656e4412d1b387f6da5e2fa934b6840711b8ead34ef3d54"],"/home/thiago/argo-workflows/ui/src/app/workflows/components/workflow-dag/graph/collapsible-node.ts":[742,1626869650833.286,"ccf6c589b5bb36a04157f724d739e745a8afab7de9b74124442f5d211ce1428b"]}
```
Comment (sarabala1979) [inline .dccache:1]: @thiago4int Can you delete this file?

File: USERS.md [hunk 2]
```diff
@@ -12,6 +12,7 @@ Currently, the following organizations are **officially** using Argo Workflows:
 
 1. [23mofang](https://www.23mofang.com/)
 1. [7shifts](https://www.7shifts.com)
+1. [4intelligence](https://4intelligence.com.br)
```
Comment (sarabala1979) [inline USERS.md:15]: can you do order move 1 up?

**Review by sarabala1979 — CHANGES_REQUESTED** [review]

File: USERS.md (hunk 2 above)
Comment (thiago4int) [reply USERS.md:14]: Done :)

File: .dccache (hunk 1 above)
Comment (thiago4int) [reply .dccache:1]: Done

**Pull request closed without merging**